docs/topics/development/debugging.rst

```diff
@@ -90,6 +88,10 @@ If clicked, it looks like:
 
 .. image:: /screenshots/django-debug-toolbar-expanded.png
 
+You must know that using the Django Debug Toolbar will slow the website quite a
+lot. You can mitigate this by deselecting the checkbox next to the ``SQL``
+panel.
```

Comment:
Might be nice to add something to the docs to say something like: "Only run this if you need it, as it makes CSP report only for local-development."